params/params.yaml

```diff
@@ -9,6 +9,7 @@ bt_navigator:
     robot_base_frame: base_link
     odom_topic: odom
     navigators: ["navigate_to_pose"]
+    # default_nav_to_pose_bt_xml: "/home/ubuntu/c1t_ws/src/c1t_bringup/behavior_trees/navigate_through_route_to_pose.xml"
```

Comment:
What's the reason for commenting this out?

Reply:
It's to show how you can override the default behavior tree. I can remove if necessary.